ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/txselection/BlockTransactionSelector.java

```diff
@@ -121,8 +123,9 @@ public BlockTransactionSelector(
     transactionSelectors = createTransactionSelectors(blockSelectionContext);
     externalTransactionSelector =
```

Comment:
since we are using `PluginTransactionSelectorFactory` and `pluginOperationTracer`, I think we can rename `externalTransactionSelector` to `pluginTransactionSelector` to make it consistent